### PR TITLE
update to PS 0.11 and other stuff

### PR DESCRIPTION
**Pull request by justinwoo**

### CODE REVIEW

**Comment by justinwoo** [comment]
I can't run the connection tests on my local but otherwise it should be good. Could you check it?

File: bower.json [hunk 1]
```diff
@@ -26,20 +26,22 @@
...
+    "purescript-prelude": "^3.1.0",
+    "purescript-foreign-generic": "^4.3.0"
   },
   "devDependencies": {
-    "purescript-spec": "^0.10.0",
-    "purescript-generics": "^3.0.0",
-    "purescript-js-date": "^3.0.0"
+    "purescript-spec": "^1.0.0",
+    "purescript-generics": "^4.0.0",
+    "purescript-js-date": "^5.1.0"
```
Comment (anttih) [inline bower.json]: This needs to be 4.0.0 still because of node-fs is not updated to 5.1.0.

**Comment by anttih** [comment]
The tests pass with js-date 4.0.0.

**Comment by justinwoo** [comment]
Ah, yeah, sorry. Realized that I was trying to update some other dependency or something.

**Comment by anttih** [comment]
Thanks!

**Pull request merged**